generator/src/generators/perfect-numbers.cr

```diff
@@ -0,0 +1,53 @@
+require "../exercise_generator"
```

Comment:
I know the file names in the generators are a bit loose but I think it should be snake case since they're crystal files. So could this be `generator/src/generators/perfect_numbers.cr`?

Reply:
Will do!